forms-flow-admin/src/containers/head.tsx

```diff
@@ -14,7 +14,8 @@ const Head = React.memo((props: any) => {
             }`}
             data-testid={`head-item-${key}`}
           >
-            <h3 onClick={item?.onClick} className="application-head">
+            <h1 className="sr-only" >{page}</h1>
```

Comment:
why we need to pass page to here actually in our application we are passing item's count @fahad-aot  please confirm it